### PR TITLE
Use Wordhat for Behat

### PR DESCRIPTION
**Pull request by ataylorme**
Drop Pantheon tests in favor of example test decoupled from our specific plugins.

Currently using a fork of WordHat, can (hopefully) drop if/when open pull requests to that project are merged.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,15 +14,15 @@
       "type": "package",
       "package": {
         "name": "pantheon-systems/wordpress",
```
Comment (greg-1-anderson) [inline composer.json]: pantheon-systems/wordpress-composer is in Packagist now; perhaps you could test with that?

**Comment by greg-1-anderson** [comment]
Awesome!

File: tests/behat-pantheon.yml [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
-       tags: "~@upstreamonly"
+    default:
       paths:
-        - ../vendor/pantheon-systems/pantheon-wordpress-upstream-tests/features/
+        - ./behat/
       contexts:
-        - PantheonSystems\PantheonWordPressUpstreamTests\Behat\MinkContext
-        - PantheonSystems\PantheonWordPressUpstreamTests\Behat\AdminLogIn
-        - PantheonSystems\PantheonWordPressUpstreamTests\Behat\ResponseHeader
+        #- FeatureContext
```
Comment (stevector) [inline tests/behat-pantheon.yml]: @ataylorme Any reason to make the file `tests/behat/bootstrap/FeatureContext.php` but then comment out this line which would load it?

File: composer.json (hunk 1 above)
Comment (stevector) [inline composer.json]: Let's do that in a follow up PR

**Comment by stevector** [comment]
Great job getting this working @ataylorme! I made a few commits adding REAMEs and tweaking the `.feature` files. I think we're just about ready to merge.

This branch now contains the README from https://github.com/stevector/example-wordpress-composer/pull/25 so I'd like to merge that first.

File: tests/behat-pantheon.yml (hunk 2 above)
Comment (ataylorme) [inline tests/behat-pantheon.yml]: @stevector the feature context was an empty example and caused the tests to fail. I've removed the file and the commented out reference.

**Comment by ataylorme** [comment]
> I made a few commits adding REAMEs and tweaking the .feature files.
Awesome - thanks @stevector. Excited to get this merged.

**Pull request merged**